src/chrome/content/rules/Sahih-Bukhari.com.xml

```diff
@@ -1,6 +1,8 @@
-<ruleset name="sahih-bukhari.com">
+<ruleset name="Sahih-Bukhari.com">
 	<target host="sahih-bukhari.com" />
 	<target host="www.sahih-bukhari.com" />
 
+	<securecookie host=".+" name=".+" />
+
```

Comment:
Generally, I would not add `<securecookie>` to existing rulesets, or the new ones, unless to fix an issue with the website.